src/App.vue

```diff
@@ -226,7 +226,7 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
   haveData = false
 
   /** The Update Current JS Date timer id. */
-  private updateCurrentJsDateId = 0
+  private updateCurrentJsDateId = null as any // NodeJS.Timeout
```

Comment:
The type is NodeJS.Timeout when running locally, or number when running in browser, so I just changed it to "any".